template-sample-app/template/src/components/common/Map.js

```diff
@@ -69,8 +83,10 @@ export function Map(props) {
     }
   };
 
+  let map;
+
   if (basemap.type === 'mapbox') {
-    return (
```

Comment:
Out of curiosity, why the refactor didn't you like the early return ?

Reply:
This is to be able to wrap it into the div with the new class without repeating this part of the code